### ISSUE TITLE
Navigation links with drop-downs are not accessible via keyboard navigation

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#533]
The three links with drop-down menus: "Uploads", "Admin", "More" are not accessible with keyboard navigation. The `tab` key skips over all three when tabbing through the menus.

We need the ability to tab into those menus, have the menus expand, and be able to select the items within. They will also need to be added to the tab index.

![links-arent-accessible](https://user-images.githubusercontent.com/45127691/102560815-29cac900-40a1-11eb-80b0-304c1d55fca0.PNG)

**Comment by librod89** [comment]
I'd like to work on this ticket.
